jobs/cloud_controller_ng/templates/bpm.yml.erb

```diff
@@ -12,6 +12,22 @@ def mount_nfs_volume!(config)
   end
 end
 
+def enable_fog_debugging!(config)
+    if p("cc.log_fog_requests")
+        case p("cc.packages.fog_connection.provider", "").downcase
+        when "azurerm"
+          config["env"]["FOG_DEBUG"] = true
+        when  "aliyun"
+          config["env"]["ALIYUN_OSS_SDK_LOG_LEVEL"] = "debug"
```

Comment:
Can you provide where this env variable is documented?  Not seeing anything about it in fog-aliyun.

Reply:
The env var of fog-aliyun is not documented. We got it from:https://github.com/aliyun/aliyun-oss-ruby-sdk/blob/d0a02321bf53798525d17f1c53ed780c2fe7437b/lib/aliyun/common/logging.rb#L47 
We've also tested it on our landscape, there were no problem.